### PR TITLE
Merge multiple changes

### PR DESCRIPTION
**Pull request by coasterteam**
This adds various things:
- LinkedList logs (default on, turn off by setting the setting `settings.UseLinkedListsInLogs` to false in Loader (massive performance improvement if on, you lose that when disabled)
- Commands `:createstat` and `:removestats` 
- RateLimiter memory leak fix

### CODE REVIEW

**Comment by Dimenpsyonal** [comment]
Screenshot 2023-11-04 at 18 05 04

How does this even happen

**Comment by Dimenpsyonal** [comment]
This is all because you did that accidental commit to master branch...

**Comment by coasterteam** [comment]
I fixed conflicts, silly github

**Comment by Dimenpsyonal** [comment]
well done

File: Loader/Config/Settings.lua [hunk 1]
```diff
@@ -291,6 +291,7 @@ settings.SystemTitle = "System Message"		-- Title to display in :sm and :bc
 
 settings.MaxLogs = 5000			-- Maximum logs to save before deleting the oldest
 settings.SaveCommandLogs = true	-- If command logs are saved to the datastores
+settings.UseLinkedListsInLogs = true -- If true then log system uses a linked-list system to store logs instead of tables - serverely improves server performance & allows for large logs with little to no impact except slightly increased memory usage
```
Comment (watameln) [inline Loader/Config/Settings.lua:294]: SPELLING MISTAKE!!!!

Comment (Dimenpsyonal) [reply Loader/Config/Settings.lua:294]: Where

Comment (coasterteam) [reply Loader/Config/Settings.lua:294]: I see it, silly me

**Pull request merged**